### DISCUSSION TITLE
README french version.

### DISCUSSION THREAD

**Discussion by KaGaster** [TeamNewPipe/NewPipe#10891] [Questions] [answered]
Hello.
A french friend of mine is interested in contributing in this project. When I read the french version of README file, I have spotted some minor mistakes and some important phrases missing(The two first phrases specifying the need of bug fixes over features proposition). If you don't mind, I can fix it and send it to you before today 5 pm.

**Comment by TobiGr** [comment, marked as answer]
Thanks, go ahead :)

**Reply by KaGaster** [reply]
Hello.
Here is the proposed new version below.
Let me know your thoughts on it.

Thank you in advance.

[README.fr.md](https://github.com/TeamNewPipe/NewPipe/files/14639811/README.fr.md)

**Reply by TobiGr** [reply]
Please open a PR

**Reply by KaGaster** [reply]
Here you go
